BringgSDK.js

```diff
@@ -1252,18 +1252,20 @@ var BringgSDK = (function () {
     var onWayPointDone = function (){
       log('way point done');
 
+      if (callbacks.driverLeftCb) {
```

Comment:
it seems like we get the whole waypoint object (looked both on the native sdk and in hagmonia-js: way_point_watch_controller line 20)